src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py

```diff
@@ -177,6 +177,18 @@ def initTest(self):
                 'desc': 'LOAD PORT TABLE FEC PATTERN FAILURE',
                 'eStr': self.defaultYANGFailure['Pattern'] + ['rc']
             },
+            'PORT_VALID_AUTONEG_TEST': {
+                'desc': 'PORT_VALID_AUTONEG_TEST no failure.',
+                'eStr': self.defaultYANGFailure['None']
+            },
+            'PORT_INVALID_AUTONEG_TEST_1': {
+                'desc': 'PORT_INVALID_AUTONEG_TEST_1 must condition failure.',
+                'eStr': self.defaultYANGFailure['InvalidValue']
+            },
+            'PORT_INVALID_AUTONEG_TEST_2': {
+                'desc': 'PORT_INVALID_AUTONEG_TEST_2 must condition failure.',
+                'eStr': self.defaultYANGFailure['Pattern']
```

Comment:
Does this one fall under pattern or InvalidValue?

Reply:
I think in this case must be 'InvalidValue' but I have test failure.
Is it problem with tests?

INFO:YANG-TEST:
------------------- Test 12: PORT_INVALID_AUTONEG_TEST_2 must condition failure.---------------------
libyang[0]: Value "260" does not satisfy the constraint "0..256" (range, length, or pattern). (path: /sonic-port:sonic-port/PORT/PORT_LIST[port_name='Ethernet8']/autoneg)
INFO:YANG-TEST:Value "260" does not satisfy the constraint "0..256" (range, length, or pattern).
ERROR:YANG-TEST: Exception >Mismatch ['Invalid value'] and Value "260" does not satisfy the constraint "0..256" (range, length, or pattern).< in /sonic/src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py:370
INFO:YANG-TEST:PORT_INVALID_AUTONEG_TEST_2 must condition failure. Failed

Reply:
I guess, for Range, libyang throws a different error, We can add a new error type: 

```
self.defaultYANGFailure = {
            'Must': ['Must condition', 'not satisfied'],
            'InvalidValue': ['Invalid value'],
            'LeafRef': ['Leafref', 'non-existing'],
            'When': ['When condition', 'not satisfied'],
            'Pattern': ['pattern', 'does not satisfy'],
            'Mandatory': ['required element', 'Missing'],
            'Verify': ['verified'],
            'Range': ['does not satisfy', 'range'],
            'None': []
        }
```

And in the test case:
```
'eStr': self.defaultYANGFailure['Range'] + ['0..256']
```

Reply:
Thanks for the hint. Added 'Range', please check.